### PR TITLE
Added post-clean script for apt upgrade

### PR DESCRIPTION
**Pull request by wizofe**
Implement a cache cleaning functionality after the installation and upgrade of `apt` via/and `dpkg`. 

It is using the `apt` `daemon` post upgrade and install process to be sure that it's called either through the `kano-updater` or via any manual package install/upgrade from the user.

INFO: Changed per @skarbat suggestion to include a diagnostic/informative message before igniting the process.

### CODE REVIEW

**Comment by wizofe** [comment]
[(2) As a user, I want the updater to autoclean old cached files [2]](https://trello.com/c/2rgwr3GT/1744-2-as-a-user-i-want-the-updater-to-autoclean-old-cached-files-2)

File: conf/apt.conf.d/99clean [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Keep the post installation environment clean
+# and completely empty the cache after upgrade/update
+# for more options see: man (5) apt.conf 
+
+DPkg::Post-Invoke { "echo Kano cache cleanup...;rm -f /var/cache/apt/archives/*.deb /var/cache/apt/archives/partial/*.deb /var/cache/apt/*.bin || true"; };
+APT::Update::Post-Invoke { "echo Kano post cleanup...;rm -f /var/cache/apt/archives/*.deb /var/cache/apt/archives/partial/*.deb /var/cache/apt/*.bin || true"; };
```
Comment (radujipa) [inline conf/apt.conf.d/99clean]: Don't these `echo`s need the string to be quoted? Also, I'd perhaps use the `logger` for this to be in the journal.

File: conf/apt.conf.d/99clean [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# Keep the post installation environment clean
+# and completely empty the cache after upgrade/update
+# for more options see: man (5) apt.conf 
+
+DPkg::Post-Invoke { "echo Kano cache cleanup...;rm -f /var/cache/apt/archives/*.deb /var/cache/apt/archives/partial/*.deb /var/cache/apt/*.bin || true"; };
```
Comment (skarbat) [inline conf/apt.conf.d/99clean]: ouch... sorry. single quotation marks for the echo literals?

File: conf/apt.conf.d/99clean (hunk 1 above)
Comment (skarbat) [reply conf/apt.conf.d/99clean]: Personally I think calling logger in the middle of an apt hook can be dangerous.

Comment (wizofe) [reply conf/apt.conf.d/99clean]: Thinking if you agree, to merge it as it is, with a simple echo to show info and only change in future if there is a need.

**Pull request merged**